### PR TITLE
DLC Quest: Add options presets to DLC Quest

### PR DESCRIPTION
**Pull request by agilbert1412**
## What is this fixing or adding?
DLC Quest did not have option presets, here are a few. All sane settings, intended for players who are not familiar enough to pick specific things, but want to give it a shot.

## How was this tested?
It's just presets

### CODE REVIEW

File: worlds/dlcquest/presets.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+from typing import Any, Dict
+
+from Options import DeathLink
```
Comment (Exempt-Medic) [inline worlds/dlcquest/presets.py:3]: This isn't used?

**Review by Exempt-Medic — APPROVED** [review]
Changes LGTM. Merged into `main` and ran a local WebHost and saw that the presets worked properly

**Review by axe-y — APPROVED** [review]
Look good

**Comment by NewSoupVi** [comment]
Has conflicts now after merging the option groups PR

**Comment by NewSoupVi** [comment]
nvm it's fine

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**